### PR TITLE
Dropout regularization dp=0.05/0.10 on 5L/256d+FourierEmbed baseline

### PR DESCRIPTION
**Pull request by morganmcg1**
## Hypothesis

The current best (PR #174, alphonse) uses 5L/256d+FourierEmbed+T_max=50 with no dropout, achieving val_abupt=6.9549%. The model may be overfitting to training geometries — especially for the high-frequency wall-shear fields (wsy=8.7345%, wsz=10.5766%) which remain far above the AB-UPT targets (3.65%, 3.63%). Light dropout regularization may smooth out overfitting in the shear channels and improve generalization, potentially closing part of the wsy/wsz gap.

**Hypothesis**: Dropout at dp=0.05 and dp=0.10 applied within the transformer feedforward/attention blocks of the 5L/256d model will reduce overfitting on wall-shear fields and lower val_abupt below the current 6.9549% baseline.

## Background

- Current best: PR #174 (alphonse), val_abupt=**6.9549%**, run `vu4jsiic`, ep~45.3
- Binding constraint: wsy=8.7345% (target 3.65%, gap 5.08pp), wsz=10.5766% (target 3.63%, gap 6.95pp)
- Wave 1 tested dropout=0.1 on a 4L/256d base (norman, run ended ep30, abupt~10.5%) — but this was on the older, weaker base. Now we have 5L/256d + FourierEmbed + T_max=50 as the proven foundation.
- Light regularization (dp=0.05) has different dynamics from medium (dp=0.10) — both are worth screening.

## Implementation

`train.py` already supports `--dropout` (passed as `model_dropout` to the model constructor). No code changes required.

**Trial A** — Light dropout dp=0.05:
```bash
cd target/ && torchrun --standalone --nproc-per-node=4 train.py \
  --model-layers 5 \
  --model-hidden-dim 256 \
  --model-heads 4 \
  --fourier-pe \
  --no-use-ema \
  --lr 3e-4 \
  --lr-cosine-t-max 50 \
  --dropout 0.05 \
  --no-compile-model \
  --wandb-group bengio-wave17-violet
```

**Trial B** — Medium dropout dp=0.10 (run in parallel with Trial A, or after Trial A ep10 result):
```bash
cd target/ && torchrun --standalone --nproc-per-node=4 train.py \
  --model-layers 5 \
  --model-hidden-dim 256 \
  --model-heads 4 \
  --fourier-pe \
  --no-use-ema \
  --lr 3e-4 \
  --lr-cosine-t-max 50 \
  --dropout 0.10 \
  --no-compile-model \
  --wandb-group bengio-wave17-violet
```

## Baseline to beat

| Metric | Current Best (val) | AB-UPT Target |
|--------|-------------------|--------------|
| `val_primary/abupt_axis_mean_rel_l2_pct` | **6.9549** | 4.51 |
| `val_primary/surface_pressure_rel_l2_pct` | 4.5644 | 3.82 |
| `val_primary/volume_pressure_rel_l2_pct` | 3.9361 | 6.08 |
| `val_primary/wall_shear_y_rel_l2_pct` | 8.7345 | 3.65 |
| `val_primary/wall_shear_z_rel_l2_pct` | 10.5766 | 3.63 |

Architecture: n_params=3,249,813 (5L/256d/4H + FourierEmbed). Confirm your run matches.

## Early stopping criteria

Kill if by epoch 5: `val_primary/abupt_axis_mean_rel_l2_pct > 13.0%`
Kill if by epoch 10: `val_primary/abupt_axis_mean_rel_l2_pct > 11.0%`
Kill if by epoch 15: `val_primary/abupt_axis_mean_rel_l2_pct > 9.5%`
Kill if by epoch 30: `val_primary/abupt_axis_mean_rel_l2_pct > 8.0%`

## Success criteria

Beat current baseline `val_primary/abupt_axis_mean_rel_l2_pct < 6.9549%` on at least one trial. Pay special attention to whether wsy and wsz improve vs baseline — those are the binding axes.

## Mandatory flags

- `--fourier-pe` is required for comparability (n_params=3,249,813 confirms FourierEmbed active)
- `--no-compile-model` is required (torch.compile/Inductor crash at validation, known bug)
- Do NOT use `--use-ema` — EMA without grad-clip causes divergence
- Do NOT use `--skip-update-grad-norm` — causes deadlock on DrivAerML

## Reporting

Report `val_primary/abupt_axis_mean_rel_l2_pct`, `surface_pressure_rel_l2_pct`, `volume_pressure_rel_l2_pct`, `wall_shear_y_rel_l2_pct`, `wall_shear_z_rel_l2_pct` at the best validation checkpoint for each trial. Include W&B run IDs.

### CODE REVIEW

**Comment by morganmcg1** [comment]
ADVISOR: @violet — This Wave 17 PR was assigned on 2026-05-02 and has had zero activity. Your pod is 1/1 READY. Please ACK this PR within the next hour and confirm you have launched your run. Post your run ID (wandb run URL or run-id hash) as a comment here once started.

If you have a blocker or question, post it now and I will respond immediately. Unacknowledged assignments will be reassigned.

**Comment by morganmcg1** [comment]
ADVISOR: Closing PR #477 — nudge posted at 03:39Z (over 2 hours ago), no ACK received. Pod confirmed as zombie in issue #466. Will reassign to @violet once human team performs SIGKILL and pod recovers.
